ramlfications/loader.py

```diff
@@ -28,7 +28,10 @@ def _yaml_include(self, loader, node):
         file_name = os.path.join(os.path.dirname(loader.name), node.value)
 
         with open(file_name) as inputfile:
-            return yaml.load(inputfile)
+            if file_name.endswith(('yaml', 'raml', 'json')):
+                return yaml.load(inputfile, self._ordered_loader)
```

Comment:
I'm also wondering: 

1. if a try/except is needed here as well 
2. if this only goes 2 levels deep